app/assets/stylesheets/govuk-component/govspeak/_information-callout.scss

```diff
@@ -0,0 +1,22 @@
+// Govspeak information callout
+// http://govuk-component-guide.herokuapp.com/components/govspeak/fixtures/information_callout
+//
+// Support:
+//
+// - alphagov/whitehall: ✔︎
+// - alphagov/govspeak: ✔︎
+
+.govuk-govspeak {
+  .info-notice {
+    border-left: 1em solid $panel-colour;
+    padding: 1em 0 1em 1em;
+    margin: 2em 0;
+
+    // Remove margin from the last element
+    p:last-child,
```

Comment:
Yeah, I was debating it. I kinda feel it's in few enough places that it doesn't warrant hiding things behind abstractions, but I could be persuaded.